responsibleai/responsibleai/_interfaces.py

```diff
@@ -20,6 +20,7 @@ class Dataset:
     class_names: List[str]
     categorical_features: List[str]
     target_column: str
+    metadata_columns: List[List]
```

Comment:
This is perhaps a place where is should be `Optional[List[str]]` ?

Reply:
This inspired me to create #1214 . Once that's merged I'll add the same annotations here, too.

Reply:
Actually it's `List[str]` (updated just now). It doesn't need to be optional for this. It can be set as `[]`